NextcloudTalk/NCChatViewController.m

```diff
@@ -3840,9 +3847,24 @@ - (void)cellWantsToDownloadFile:(NCMessageFileParameter *)fileParameter
 
 - (void)cellHasDownloadedImagePreviewWithHeight:(CGFloat)height forMessage:(NCChatMessage *)message
 {
+    BOOL isAtBottom = [self shouldScrollOnNewMessages];
+
     message.file.previewImageHeight = height;
+
+    [CATransaction begin];
+    [CATransaction setCompletionBlock:^{
+        dispatch_async(dispatch_get_main_queue(), ^{
+            // make sure we're really at the bottom after updating a message since the file previews could grow in size if they contain a media file preview, thus giving the effect of not being at the bottom of the chat
+            if (isAtBottom) {
```

Comment:
```suggestion
            if (isAtBottom && !self.tableView.isDecelerating) {
```

If we scrolled with a finger, we should not try to scroll back to the bottom. That can happen if there are a lot if pictures as last messages and one tries to slowly scroll upwards and a new image preview is loaded,  but we're not above the threshold to not scroll.